components/CustomSelect.tsx

```
@@ -6,11 +6,13 @@ import AntDesign from '@expo/vector-icons/AntDesign';
 
 import Text from './CustomText';
 
-interface Props {
+type Items = Array<{label: string; value: string}>;
+
+export interface Props {
   onChange: (data: any) => any;
   value: any;
   styles?: Object;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)